site/_includes/layouts/collection-landing.njk

```diff
@@ -0,0 +1,31 @@
+{% extends "layouts/base.njk" %}
+
+{% from 'macros/cards/hero-card.njk' import heroCard with context %}
+{% from 'macros/icon.njk' import icon with context %}
+{% from 'macros/cards/blog-card.njk' import blogCard with context %}
+
+{% block content %}
+    <div class="measure-1280 width-full center-margin pad-400">
+      {{ heroCard(title, description) }}
+
+      <div class="pad-top-400">
+        {{ content | safe }}
+      <div>
+
+      <div class="blog-grid pad-top-400">
+        {% for post in collections.tags[collection_tag].posts.en %}
+          {% if loop.index0 < 6 %}
+            {{ blogCard(post) }}
+          {% endif %}
+        {% endfor %}
+      </div>
+
+      <nav class="pagination gap-top-300 pad-left-400 pad-right-400">
+        <span></span>
```

Comment:
Just curious—what's up with the empty `<span/>`?

Reply:
Because I'm re-using .pagination class here, it expects to have 2 children in the grid. I'll add a comment to make it clear.